tictactoe/__init__.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
```

Comment:
yes, but `player.py` also needs `Cell.EMPTY` check, so to prevent circular import we need it to be stored in a "shared" place or having an import within a `get_turn` method which I think would be a bad practice. I might also move this to separate `cell.py` module in case `__init__.py` is not the right place.

Reply:
Yeah, I missed that it was also used in `player.py` (left the comment before I had read that file)